README.md

```diff
@@ -178,26 +174,28 @@ export default {
 }
 ```
 
-### `client.js`
-
-export configuration for [`feathers` client](https://docs.feathersjs.com/api/client.html)
-
-- `services`: export an array of functions that will be run with [`client.configure(plugin)`](https://docs.feathersjs.com/api/application.html#configurecallback)
+#### `client`
+configuration for [`feathers` client](https://docs.feathersjs.com/api/client.html), as an object with `services` and `config` properties:
+- `services`: an array of functions that will be run with [`client.configure(plugin)`](https://docs.feathersjs.com/api/application.html#configurecallback)
+- `apiUrl`: the url of the api server for the client to connect to (normally this can be extracted from your `config`)
 
+example:
 ```js
 // client.js
 export default {
-  services: []
+  services: [
+    authentication
+  ],
+  apiUrl: config.api.url
```

Comment:
given that we prescribe the shape of `config.api.url`, would it be more friendly to just take in the whole config?

Reply:
👍 as discussed